docs/data/routes/docs/fundamentals/services/tracking/en.md

```diff
@@ -75,13 +75,8 @@ trackingApi
 
 To see the data pushed by your tracking code, the easiest way is to use the [xProfile](https://doc.sitecore.net/sitecore_experience_platform/digital_marketing/experience_profile/experience_profile). Sitecore does not store or index the tracked data into xProfile until after the user session ends; thus it is essential to end your session to see the test data. The styleguide tracking example includes an example button that invokes the JSS tracker session abandon method that will cause an instant data flush. This can also be manually invoked on your Sitecore instance by visiting `/sitecore/api/jss/track/flush`.
 
-Because tracking data is submitted directly to a Sitecore service, you cannot use the JSS tracking API in [disconnected mode](/docs/fundamentals/application-modes). There are also limitations when in Connected Mode due to cookie data; see below.
-
-> Not seeing any data in xProfile after ending your session? There are several possible causes:
->
-> * Ensure you have an XP-enabled Sitecore license (XM cannot use tracking)
-> * Your analytics ID has been flagged as robot traffic and dropped. Check the value of the `SC_ANALYTICS_GLOBAL_COOKIE` cookie; if it ends with `|False`, your analytics ID is flagged as a robot. A common cause of this is using [connected mode](/docs/fundamentals/application-modes) with the tracker; unless a pre-existing analytics cookie is in your browser for the layout service host from _integrated mode_, the robot detection process will not operate in connected mode and you will be flagged a robot. Visit the app in integrated mode, clear your analytics cookie, refresh, and move your mouse to be redetected as a human visitor.
-> * Your analytics ID is not [identified](https://doc.sitecore.net/developers/xp/tracking-and-session/tracker/tracking-contacts/identification/index.html), and [indexing anonymous contacts](https://doc.sitecore.net/developers/xp/xconnect/xconnect-search-indexer/enable-anonymous-contact-indexing.html) is disabled, which will prevent your un-identified data from showing up in xProfile.
+> * Because tracking data is submitted directly to a Sitecore service, you cannot use the JSS tracking API in [disconnected mode](/docs/fundamentals/application-modes).
```

Comment:
@illiakovalenko Seems like these disclaimers should be at the top of the page, along [with the notes about secure cookies](https://github.com/Sitecore/jss/blob/master/docs/data/routes/docs/fundamentals/services/tracking/en.md). And realizing we're missing those on dev branch, so we should do a merge from master.